.agents/merge-workflow.sh

```
@@ -0,0 +1,345 @@
+    local title="$4"
+    local description="$5"
+    local priority="$6"
+    
+    echo -e "${PURPLE}📝 Creating PR for $todo_id...${NC}"
+    cd "$worktree_path"
+    
+    # Push branch to origin
+    git push -u origin "$branch_name"
+    
+    # Create PR body with structured information
+    local pr_body="## $todo_id: $title
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)